kornia/geometry/transform/pyramid.py

```diff
@@ -410,3 +409,24 @@ def build_laplacian_pyramid(
         laplacian_pyramid.append(laplacian)
     laplacian_pyramid.append(gaussian_pyramid[-1])
     return laplacian_pyramid
+
+
+def upscale_double(x):
+    r"""Upscale image by the factor of 2, even indices maps to original indices.
+
+    Odd indices are linearly interpolated from the even ones.
+
+    Args:
+        x : input image.
+    Shape:
+        - Input: :math:`(B, C, H, W)`
+        - Output :math:`(B, C, H, W)`
+    """
+    B, CH, H, W = x.shape
+    upscaled = torch.zeros((B, CH, H * 2, W * 2))
+    upscaled[:, :, ::2, ::2] = x
```

Comment:
this isnt an option here?
```suggestion
    upscaled[..., ::2, ::2] = x
```

Reply:
+1 for (*,H,W) unless there's objection

Reply:
implemented